_studio/shared/umc/codec/h265_dec/src/umc_h265_heap.cpp

```diff
@@ -343,8 +343,7 @@ uint8_t * CumulativeArraysAllocation(int n, int align, ...)
     int cumulativeSize = 0;
     for (int i = 0; i < n; i++)
     {
-        void * ptr = va_arg(args, void *);
-        ptr; // just skip it
+        va_arg(args, void *);
```

Comment:
Maybe we will remove the whole function if we don't use it? Why you have left it?

Reply:
I plane to remove all unused functions/methods in the next PR (for keep structure: PR - type of issue, commit - component)